.circleci/config.yml

```diff
@@ -547,6 +590,13 @@ workflows:
             branches:
               only: master
 
+      - build_release_candidate_docker_image:
+          python_version: '3.7'
```

Comment:
any reason to restrict this?

Reply:
No intention here. This is just the test image for our round of internal tests; we can add other versions now or when we add the CI job to publish to PyPi.